src/utilities/FormManager.ts

```diff
@@ -78,9 +78,8 @@ class FormManager {
     return getFieldSetsFields(this.args.fieldSets);
   }
 
-  public getDefaultValue (fieldConfigPartial: IFieldConfigPartial) {
+  public getDefaultValue (fieldConfig: IFieldConfig) {
```

Comment:
What has changed to ensure that these are `fieldConfig`s, not `fieldConfigPartial`s? Have these always been full `fieldConfig`s?

Reply:
It looks like it was never actually called with a Partial. I was just leaving it open to make it as usable as possible but it was wasting cycles filling in non-partials all the time.